### PR TITLE
docs: update path for zh-hans docs

### PR DESCRIPTION
**Pull request by yzeng25**
Changes:

As https://github.com/apache/apisix-website/pull/637 updates, now blog pages are split into `blog` and ``/zh/blog/. `blog` is for blogs in English, and `/zh/blog/` is for blogs in zh-hans. 

So I moved some blogs in zh-hans to the corresponding path and deleted the `draft` tag to let them show up in `/zh/blog/` . I was not aware of this change when I created https://github.com/apache/apisix-website/pull/648, and created some Chinese blogs in the English directory. 

This PR moved these Chinese docs to the right directory, and fixed a broken link issue in release notes for 2.10.0

### CODE REVIEW

File: website/blog/2021-09-29-release-apache-apisix-2.10-en.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ In addition to the Python Plugin Runner, our community partners are also develop
 
 ## Download
 
-In addition to the above new features and components, the Apache APISIX 2.10.0 release introduces more than a dozen new features and plugins, for details please see the [Change log](https://github.com/apache/apisix/blob/release/2.10/) corresponding to this release CHANGELOG.md#2100).
+In addition to the above new features and components, the Apache APISIX 2.10.0 release introduces more than a dozen new features and plugins, for details please see the [Change log](https://github.com/apache/apisix/releases/tag/2.10.0) corresponding to this release.
```
Comment (bzp2010) [inline website/blog/2021-09-29-release-apache-apisix-2.10-en.md]: Should be "https://github.com/apache/apisix/blob/release/2.10.0/CHANGELOG.md#2100" here?
Only links are provided on the release log without actual content.

File: website/i18n/zh/docusaurus-plugin-content-blog/2021-09-29-release-apache-apisix-2.10.md [hunk 2]
```diff
@@ -106,7 +106,7 @@ Python 是一门有着深厚群众基础的编程语言，一直以容易上手
 
 ## 下载
 
-除了上述新功能和组件外，Apache APISIX 2.10.0 版本还引入了十余个新功能和插件，详情请查看本次发布对应的 [Change log](https://github.com/apache/apisix/blob/release/2.10/CHANGELOG.md#2100)。
+除了上述新功能和组件外，Apache APISIX 2.10.0 版本还引入了十余个新功能和插件，详情请查看本次发布对应的 [Change log](https://github.com/apache/apisix/releases/tag/2.10.0)。
```
Comment (bzp2010) [inline website/i18n/zh/docusaurus-plugin-content-blog/2021-09-29-release-apache-apisix-2.10.md]: ditto

File: website/blog/2021-09-29-release-apache-apisix-2.10-en.md (hunk 1 above)
Comment (yzeng25) [reply website/blog/2021-09-29-release-apache-apisix-2.10-en.md]: Yes, thanks!

**Comment by yzeng25** [comment]
close due to path modification

**Pull request closed without merging**